src/Event.cpp

```diff
@@ -25,38 +25,90 @@ void Event::add_value(std::string name, std::string&& arg) {
     string_values_.emplace(name, arg);
 }
 
+//get_value
+//Params: name of event, address to store result at, bool for error checking. if true and an error occurs then exit program
 template<>
+<<<<<<< HEAD
```

Comment:
found the merge conflict, not gonna review this file till you fix the merge conflicts